### PR TITLE
virttest.qbuses: enable "serial" for usb devices

### PR DESCRIPTION
**Pull request by hereischen**
Added param "serial" for usb devices, so that usb devices can
be identified in the guest.

id: 1482404
Signed-off-by: Haotong Chen <hachen@redhat.com>

### CODE REVIEW

File: virttest/qemu_devices/qcontainer.py [hunk 1]
```diff
@@ -1181,7 +1183,8 @@ def usb_by_params(self, usb_name, params):
                                      params.get("usb_type"),
                                      params.get("usb_controller"),
                                      params.get("bus"),
-                                     params.get("port"))
+                                     params.get("port"),
+                                     params.get("serial"))
```
Comment (luckyh) [inline virttest/qemu_devices/qcontainer.py]: Suggest using `usbdev_serial`.

Comment (hereischen) [inline virttest/qemu_devices/qcontainer.py]: updated. thanks.

File: virttest/qemu_devices/qcontainer.py [hunk 2]
```diff
@@ -1141,14 +1141,15 @@ def usbc_by_params(self, usb_name, params, pci_bus={"aobject": "pci.0"}):
...
-                         port=None):
+                         port=None, serial=None):
         """
         Creates usb-devices by variables.
         :param usb_name: usb name
         :param usb_type: usb type (usb-tablet, usb-serial, ...)
         :param controller_type: type of the controller (uhci, ehci, xhci, ...)
         :param bus: the bus name (my_bus.0, ...)
         :param port: port specifiacation (4, 4.1.2, ...)
+        :param serial: serial specifiacation (1234, d1, ...)
```
Comment (lmr) [inline virttest/qemu_devices/qcontainer.py]: typo, `specification`

**Comment by lmr** [comment]
Please rebase against the latest master, I've fixed Travis CI

File: virttest/qemu_devices/qcontainer.py (hunk 2 above)
Comment (hereischen) [inline virttest/qemu_devices/qcontainer.py]: fixed , thanks

**Comment by xutian** [comment]
Looks OK to me.

**Comment by luckyh** [comment]
Thanks all.

**Pull request merged**